javatests/google/registry/rde/imports/RdeDomainImportActionTest.java

```diff
@@ -135,6 +148,151 @@ public void test_mapreducePublishesToDns() throws Exception {
     assertDnsTasksEnqueued("example1.test");
   }
 
+  /**
+   * Verifies the state of an imported pending transfer before and after implicit server approval
+   */
+  @Test
+  public void testMapreducePendingTransferServerApproval() throws Exception {
```

Comment:
FWIW, there are additional tests that would be good to have either here or in XjcToDomainResourceConverterTest, specifically:

- autorenew grace period subsuming behavior (I can take care of this one)
- 10-year capping case, where transfer exDate isn't a year multiple after current exDate
- domain with previous (non-pending) transfer data - domain_fragment_transferPeriod.xml actually includes this, but the TransferData fields themselves are not checked in the test, only the grace period